src/doc/src/reference/workspaces.md

```diff
@@ -30,6 +33,10 @@ In the `Cargo.toml`, the `[workspace]` table supports the following sections:
 * [`[replace]`](overriding-dependencies.md#the-replace-section) --- Override dependencies (deprecated).
 * [`[profile]`](profiles.md) --- Compiler settings and optimizations.
 
+## Creating a Workpaces
+
+todo!：Added example of creating a workspace.
```

Comment:
btw this is a reference-level page.  I feel like this would probably belong in the Guide section.  However, I'm not quite too sure what all we'd want to cover in a guide-level section on workspaces.  Might be good to discuss this in the issue first.

Reply:
Ok, about this part I will first submit a preliminary idea for discussion in issue.